github/actions/config.go

```diff
@@ -96,3 +99,16 @@ func (c *GitHubConfig) GitHubAPIURL(path string) *url.URL {
 
 	return result
 }
+
+func isHostedGitHubURL(u *url.URL) bool {
+	forceGhes := os.Getenv("GITHUB_ACTIONS_FORCE_GHES")
+
+	if len(forceGhes) > 0 {
```

Comment:
If it is only important that the env is set, we can leverage `os.LookupEnv`, although this function returns true if env is set but value is empty